### ISSUE TITLE
Fix Request for pmd7 AvoidHardcodedConnectionConfig false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#398]
```
public final class InputNormalizer {
    private static final int MAXIMUM_LENGTH_FOR_NAME = 30; // <--- false positive
}
```
